extension/popup/popup.html

```diff
@@ -14,6 +14,10 @@
     <script src="../global_files/redux.js"></script>
     <script src="../global_files/react-redux.js"></script>
     <script src="../global_files/redux-webext.js"></script>
+    <script src="../global_files/jquery-3.2.1.slim.js"></script>
+    <script src="../global_files/popper.js"></script>
+    <script src="../global_files/bootstrap/util.js"></script>
+    <script src="../global_files/bootstrap/dropdown.js"></script>
```

Comment:
There were three ways to approach dropdowns:

1. Do it myself (no thanks).
2. Use react-bootstrap (tried; only supports Bootstrap 3).
3. Do it how Bootstrap wants it and include jQuery/Popper/etc.

It's a bit of a bummer since mixing jQuery and React can be very bad, but since we don't actually use jQuery directly in the React code, it seems okay: React is always rendering the same elements, Bootstrap-via-jQuery is just noodling some of the styles. We don't even have to do weird stuff with refs!